internal/app/handler/dataplane/documentstorage/mongodb/mongodb.go

```diff
@@ -153,6 +153,27 @@ func (db *MongoDB) CreateInsight(insight *documentstorage.Insight) error {
 	return nil
 }
 
+//CreateModuleLogs creates an insights document
+func (db *MongoDB) CreateModuleLogs(logs *documentstorage.ModuleLogs) error {
+	logs.Context.DocumentType = common.ModuleLogsDocType
+	b, err := JSONMarshal(*logs)
```

Comment:
Should we split the logs into fixed size chunks for marshalling and transfer?

Reply:
For now this will stay, moving to uploading the blogs to blob and sharing a SAS url in the logs